tests/tests/test_passbands/test_passbands.py

```diff
@@ -0,0 +1,42 @@
+import phoebe
+from phoebe import u
+import os
+
+
+path = os.path.dirname(__file__)
+
+
+def test_compute(atm_grids_available=False, atm_path=None, atms=['ck2004', 'phoenix', 'tmap_sdO', 'tmap_DA', 'tmap_DAO', 'tmap_DO']):
```

Comment:
we could also import `atms` here to ensure any ones added in the future get coverage